package.json

```diff
@@ -57,13 +57,18 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "storybook": "^7.6.6",
-    "styled-components": "^6.1.2",
     "typescript": "^5.3.3",
     "vite": "^5.0.10",
     "vite-plugin-dts": "^3.6.4",
     "vitest": "^1.1.0"
   },
   "peerDependencies": {
     "react": "^18.2.0"
+  },
+  "dependencies": {
+    "@emotion/babel-plugin": "^11.11.0",
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@vitejs/plugin-react": "^4.2.1"
```

Comment:
These should be `devDependencies` instead.

`pnpm remove @emotion/babel-plugin @emotion/react @emotion/styled @vitejs/plugin-react`

`pnpm i -D @emotion/babel-plugin @emotion/react @emotion/styled @vitejs/plugin-react`